tutorial04_first_window/app/src/main/jni/VulkanMain.cpp

```diff
@@ -207,6 +207,17 @@ void CreateSwapChain() {
   LOGI("<-createSwapChain");
 }
 
+void DeleteSwapChain(void) {
```

Comment:
The `DeleteSwapChain` under the `CreateSwapChain` as all the other "Create..." have the "Delete..." under it